freqtrade/data/dataprovider.py

```diff
@@ -92,7 +92,7 @@ def historic_ohlcv(
                 'timerange') is None else str(self._config.get('timerange')))
             # Move informative start time respecting startup_candle_count
             timerange.subtract_start(
-                timeframe_to_seconds(str(timeframe)) * self._config.get('startup_candle_count', 0)
+                self.get_required_startup_seconds(str(timeframe))
```

Comment:
the explicit str is unnecessary here.
we expect "5m" for example - if it ain't a string, it'll fail anyway - the only difference will be if it fails here - or from within timeframe_to_seconds.

Reply:
Hah, I wondered the same thing but I just kept that code unchanged - unsure why it is originally doing that. Any ideas why?

Reply:
not really - but code changes over time - so most likely, this was initially a non-str input ...

Reply:
leave the `str()` there for now - it's needed for mypy.